### PR TITLE
[MM-30808] - Switch creation of users to use app method vs client method

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR switches the underlying method used to create users using `\test users` command, to use app method vs client method

#### Ticket Link
[https://mattermost.atlassian.net/browse/MM-30808?atlOrigin=eyJpIjoiMDAxNjAwNzcxNzgyNDJiYTgwOTU2ODgzNDQ5MzBmOWQiLCJwIjoiaiJ9](https://mattermost.atlassian.net/browse/MM-30808?atlOrigin=eyJpIjoiMDAxNjAwNzcxNzgyNDJiYTgwOTU2ODgzNDQ5MzBmOWQiLCJwIjoiaiJ9)
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: app/slashcommands/auto_constants.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	UserPassword         = "passwd"
+	UserPassword         = "Usr@MMTest123"
```
Comment (crspeller) [inline app/slashcommands/auto_constants.go:12]: Why change the password?

**Review by crspeller — APPROVED** [review]

File: app/slashcommands/auto_constants.go (hunk 1 above)
Comment (AGMETEOR) [reply app/slashcommands/auto_constants.go:12]: It was failing and returning an error saying password needs to contain some characters. I traced it to a password validation in the code

**Review by stevemudie — APPROVED** [review]
LGTM

**Comment by ethervoid** [comment]
@AGMETEOR what was causing the problem?

**Comment by AGMETEOR** [comment]
> @AGMETEOR what was causing the problem?

https://mattermost.atlassian.net/browse/MM-27412

**Review by ethervoid — APPROVED** [review]
LGTM! Great work @AGMETEOR !

**Pull request merged**